### DISCUSSION TITLE
CSV Import [May 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2245] [Discord Threads]
> 🙋‍♂️ **kenmt**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### No limit as such. If you face any issue, let us know  
#  
> 🙋‍♂️ **kenmt**  
#### 🤷‍♂️  
![image0](https://cdn.discordapp.com/attachments/974116150453608488/974125097478676490/Screen_Shot_2022-05-11_at_9.45.10_PM.png)  
#  
> 🙋‍♂️ **wkw**  
#### please create an issue with some details such as some logs  
#  
> 🙋‍♂️ **wkw**  
#### probably some data types don't match  
#  
> 💁 **o1lab**  
#### csv is in alpha state and we will be improving ... pls raise issue as Wing has mentioned  
#  
> 🙋‍♂️ **grndstt**  
#### got this when the id field was not mapped  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/974116150453608488)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
